### DISCUSSION TITLE
Do custom `headers()` apply to `next export`-generated assets?

### DISCUSSION THREAD

**Discussion by thiskevinwang** [vercel/next.js#27289] [Help]
I'm reading through **[next.config.js/headers](https://nextjs.org/docs/api-reference/next.config.js/headers)** and wondering, do these headers still apply when you are running `next export` and serving those static assests?

My understanding is: **no, these headers do not apply to [next export](https://nextjs.org/docs/advanced-features/static-html-export)ed assets** (since there is no longer a `nextjs` server running, which normally applies these headers), but I wanted to double check my understanding, since it's not explicitly noted as a _caveat_ on the docs.

**Comment by IshanKBG** [comment]
Well no but you can see [exportPathMap](https://nextjs.org/docs/api-reference/next.config.js/exportPathMap)
